src/rt/backtrace/dwarf.d

```diff
@@ -14,6 +14,7 @@ module rt.backtrace.dwarf;
 
 version(CRuntime_Glibc) version = linux_or_freebsd;
 else version(FreeBSD) version = linux_or_freebsd;
+else version(DragonFlyBSD) version = linux_or_freebsd;
```

Comment:
As long as we're here, can you change this to `glibc_or_bsd`?  This name has annoyed me for awhile.

Reply:
Used "glibc_or_bsdlibc" instead (thought that would be a better fit, and also usable by netbsd/openbsd when needed), if that is ok with you.